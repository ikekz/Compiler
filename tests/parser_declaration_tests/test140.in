function f(a: array of integer) : integer;
begin
end;
begin
end.