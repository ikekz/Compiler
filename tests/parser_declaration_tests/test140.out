integer        type           integer
double         type           double
char           type           char
f              function       return type: 
                              integer
                              args:
                              a              valueparam     {open} array of integer
                              locals:
                              none
